packages/server-ct/index.ts

```diff
@@ -27,7 +27,11 @@ export const start = async (projectRoot: string, args: Record<string, any>) => {
 
     return openProject.create(projectRoot, args, options)
     .then((project) => {
-      return openProject.launch(browser, spec)
+      return openProject.launch(browser, spec, {
+        onBrowserClose: () => {
+          process.exit()
```

Comment:
Looks good to me. I might've added a CT specific debug log, but it's not a deal breaker.  The current DEBUG logs look like this:

```
  cypress:launcher chrome exited: { code: 0, signal: null } +5s
  cypress:server:browsers killing browser process +4s
  cypress:server:browsers:chrome closing remote interface client +4s
✨  Done in 10.87s.
```

Reply:
we already have a debugger...  like `import { debug as debugFn } from debug` and then `const debug = debugFn('cypress:server-ct:browsers')` yknow?

Reply:
I'm fine to keep it as a callback.

Reply:
The event emit would still be emitted in the callback.